frontend/ui/lib/widgets/trip_view_widget.dart

```diff
@@ -14,26 +14,30 @@ class _TripWidgetState extends State<TripViewWidget> {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-          border: Border.all(
-            color: Colors.black,
+    return CustomScrollView(
+      slivers: [
+        SliverAppBar(
+          pinned: true,
+          floating: true,
+          snap: true,
+          expandedHeight: 250.0,
+          flexibleSpace: FlexibleSpaceBar(
+            title: Text('Name of the Trip'),
+            background: Image.network(
+              'https://www.gannett-cdn.com/presto/2019/02/01/USAT/2af52e69-3fd1-4438-99d7-487a9b51d03c-GettyImages-878868924.jpg',
+              fit: BoxFit.cover,
+            ), //Everybackground is seattle
           ),
-          borderRadius: BorderRadius.all(Radius.circular(5))),
-      margin: const EdgeInsets.all(15.0),
-      padding: const EdgeInsets.all(10.0),
-      child: SingleChildScrollView(
-        child: Column(
-          children: [
-            Text("Name of the Trip"),
-            PlaceBlockWidget("Place 1"),
-            PlaceBlockWidget("Place 2"),
-            PlaceBlockWidget("Place 3"),
-            PlaceBlockWidget("Place 4"),
-            PlaceBlockWidget("Place 5"),
-          ],
         ),
-      ),
+        SliverList(
```

Comment:
I played around with this and it feels a little weird having nested lists like this (the sublists scroll before the overall list scrolls). It might make more sense to have these inner lists be fully expanded (constructed with a delegated list builder) and have only the outer view scroll. Again, not necessarily mandatory here, but I wanted to bring this to your attention.

Reply:
Chris will be helping me make a better feeling inner scroll for the places. I also felt as if it was close but not exactly what we wanted but I had problems trying to implement the scrolling functionality that I wanted.